HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -146,7 +146,7 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
 
     // Separate SBATCH options from commands
     job.commands = editor.getValue();
-    var other = editor.getValue().split("\n");
+    var other = editor.getValue().split(/\r?\n/);
```

Comment:
Oh, yeah.  Remove them everywhere!  You should really just remove them from the editor contents, then move on saving the commands everywhere.